### ISSUE TITLE
Value Conversion Error importing automation

### ISSUE THREAD

**Issue by jbnitorum** [PrefectHQ/terraform-provider-prefect#331]
I am getting the following error when trying to import an automation:

```
╷
│ Error: Value Conversion Error
│ 
│ An unexpected error was encountered trying to build a value. This is always
│ an error in the provider. Please report the following to the provider
│ developer:
│ 
│ Received null value, however the target type cannot handle null values. Use
│ the corresponding `types` package type, a pointer type or a custom type
│ that handles null values.
│ 
│ Path: trigger
│ Target Type: resources.TriggerModel
│ Suggested `types` Type: basetypes.ObjectValue
│ Suggested Pointer Type: *resources.TriggerModel
╵
```

**Comment by jbnitorum** [comment]
Sure...

```
{
  "type": "event",
  "match": {
    "prefect.resource.id": "prefect.flow-run.*"
  },
  "match_related": {
    "prefect.resource.id": [
      "prefect.tag.Tag1",
      "prefect.tag.Tag2"
    ],
    "prefect.resource.role": "tag"
  },
  "after": [
    "prefect.flow-run.Failed",
    "prefect.flow-run.Cancelled",
    "prefect.flow-run.Crashed",
    "prefect.flow-run.Completed"
  ],
  "expect": [
    "prefect.flow-run.*"
  ],
  "for_each": [
    "prefect.resource.id"
  ],
  "posture": "Proactive",
  "threshold": 1,
  "within": 30
}
```

**Comment by jbnitorum** [comment]
Any timeline for a fix on this?

**Comment by parkedwards** [comment]
hey @jbnitorum - we've got a fix for this in #336 , and will aim to release this fix tonight

**Comment by parkedwards** [comment]
@jbnitorum give this a whirl when this release lands

https://github.com/PrefectHQ/terraform-provider-prefect/releases/tag/v2.13.2
